pkg/visor/rpc.go

```diff
@@ -22,8 +22,12 @@ import (
 const (
 	// RPCPrefix is the prefix used with all RPC calls.
 	RPCPrefix = "app-visor"
-	// HealthTimeout defines timeout for /health endpoint calls.
+	// HealthTimeout defines timeout for /health endpoint calls done from hypervisor.
 	HealthTimeout = 5 * time.Second
+	// InnerHealthTimeout defines timeout for /health endpoint calls done from visor.
+	// We keep it is less than the `HealthTimeout`, so that the outer call would
+	// definitely complete.
+	InnerHealthTimeout = 4 * time.Second
```

Comment:
Should probably be reduced to 3 seconds just to be sure. A DMSG roundtrip can take up to 1.2 seconds in certain circumstances so there should be some more space imo.